lib/mobility-core/src/Kernel/Utils/DatastoreLatencyCalculator.hs

```diff
@@ -31,6 +38,12 @@ withTime storeType operationName operation = do
   res <- operation
   atime <- getCurrentTime
   let latency = diffUTCTime atime btime
-  logTagInfo (storeType <> ":" <> operationName) $ show latency
-  addDatastoreLatency storeType operationName latency
+  enableRedisLatencyLogging <- asks (.enableRedisLatencyLogging)
+  if enableRedisLatencyLogging
```

Comment:
you can use `when` instead of `if then.. else..`:
`when enableRedisLatencyLogging $ logTagInfo (storeType <> ":" <> operationName) (show latency)`